candle/timetable/render.py

```diff
@@ -1,25 +1,42 @@
+from typing import Optional, Set
+
 from flask import render_template
 from flask_login import current_user
 from sqlalchemy.orm import joinedload, contains_eager
 
-from candle import db
-from candle.models import Lesson, Subject, Room, LessonType
+from candle.models import Lesson, Subject
 from candle.timetable.layout import Layout
 
 
-def render_timetable(title, lessons, editable=True, **kwargs):
-    lessons = lessons.join(Lesson.subject).options(joinedload(Lesson.room), joinedload(Lesson.type),
-                                                   joinedload(Lesson.teachers),
-                                                   contains_eager(Lesson.subject)).order_by(Lesson.day, Lesson.start,
-                                                                                            Subject.name).all()
-    t = Layout(lessons)
+def render_timetable(
+    title, lessons, highlighted_lesson_ids: Optional[Set[int]], editable=True, **kwargs
+):
+    lessons = lessons.join(Subject).order_by(Lesson.day, Lesson.start, Subject.name).all()
```

Comment:
Asi zlý rebase do mastra: dvakrát za sebou robíš `lessons = lessons.(...).all()` a hádže mi to TypeError.

Reply:
Diky, hej, zle som to rebasol